mixed_tabs_and_spaces.py

```diff
@@ -2,6 +2,6 @@ def square(x):
     sum_so_far = 0
     for counter in range(x):
         sum_so_far = sum_so_far + x
-	return sum_so_far  # noqa Python 3 will raise a TabError here
+	return sum_so_far  # noqa: E999 # pylint: disable=bad-indentation Python 3 will raise a TabError here
```

Comment:
`# pylint: disable=bad-indentation` doesn't suppress the warning, instead, `# pylint: disable=mixed-indentation` works for me

Reply:
Also could you please replace `counter` with `_` as it's an unused variable and pylint is raising a warning about it :sweat_smile: